content/blog/2021/announcing-tickets.md

```diff
@@ -0,0 +1,35 @@
+Title: PyCon India 2021 tickets are live
+Date: 2021-054-08 12:00
+Category: 2021
+Slug: announce-ticket-sales
+Summary: Tickets for the conference are live! Grab those limited Early Bird tickets.
+
+PyCon India is a volunteer-driven conference where Python communities across the nation come together to meet, discuss, and learn about their favorite aspects of Python. It's the country's largest premier conference for the Python programming language, unparalleled in the opportunities, knowledge, and networking potential.
+
+The conference will be having a packed schedule of workshops, keynotes, talks, and dev-sprints strung together through x parallel tracks. Any person who has a conference ticket is also eligible to attend the Devsprints.
+
+We have started with the ticket sales of our mega event and the details for the same are available on [PyCon India's official website](https://in.pycon.org/). To make the event inclusive and accessible to all, we have kept the ticket prices to a bare minimum, namely.
+
+**Rush to grab your early bird tickets now from [Townscript]()!**
+
+## Types of Tickets
+
+## Early Bird
+
+An Early bird ticket reserves your seat for 2 days of the conference and Devsprints, with the pricing of INR x (GST excluded), the sale of early bird tickets ends in 5 days after their release or when the sale of tickets has been completed, whichever comes earlier.
+
+## Regular
+
+A regular ticket reserves your seat for 2 days of the conference and Devsprints, with the pricing of INR x (GST excluded), the sale of regular tickets begins after the sale of early bird tickets ends till the sale of tickets has been completed.
+
+## Late Bird
+
+A late bird ticket reserves your seat for 2 days of the conference and Devsprints, with the pricing of INR x (GST excluded), the sale of the late bird begins after the sale of regular tickets ends till the ticket sales for the conference close. These are unlimited in number.
+
+## Contributor
+
+A Contributor ticket is for anyone to purchase who would like to support the conference in the years to come. The contributor ticket reserves your seat for 2 days of the conference and Devsprints, with the pricing of INR x (GST excluded), the sale of these tickets starts right from day 1 and are unlimited in number.

```

Comment:
Need minimum price for contributor ticket

Reply:
2500